app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt

```diff
@@ -0,0 +1,89 @@
+package com.mapbox.navigation.examples.preview.dropinui
+
+import android.os.Bundle
+import android.view.Gravity
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.dropin.MapboxExtendableButtonParams
+import com.mapbox.navigation.examples.preview.R
+import com.mapbox.navigation.examples.preview.databinding.MapboxActivityCustomizeActionButtonsBinding
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+class CustomActionButtonsActivity : AppCompatActivity() {
+
+    private lateinit var binding: MapboxActivityCustomizeActionButtonsBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = MapboxActivityCustomizeActionButtonsBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.navigationView.api.routeReplayEnabled(true)
+
+        binding.navigationView.customizeViewStyles {
+            val layoutParams = LinearLayout.LayoutParams(
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+                ViewGroup.LayoutParams.WRAP_CONTENT
+            )
+            audioGuidanceButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomAudioGuidanceButtonCircular,
+                LinearLayout.LayoutParams(
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                    ViewGroup.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    marginStart = 10
+                }
+            )
+            cameraModeButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomCameraModeButtonCircular,
+                LinearLayout.LayoutParams(
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                    ViewGroup.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    topMargin = 20
+                    marginStart = 10
+                }
+            )
+            recenterButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomRecenterButtonCircular,
```

Comment:
```suggestion
                R.style.MapboxStyleRecenterButton_Circle,
```

Reply:
Reused for action buttons. Cannot be done for `Start Navigation`, `Route Preview` and `End Navigation` buttons.